### PR TITLE
Remove recoilState to use search query

### PR DESCRIPTION
**Pull request by chenz4027**
Signed-off-by: Maggie Chen <magchen@redhat.com>
https://github.com/stolostron/backlog/issues/22969

image

### CODE REVIEW

File: frontend/src/lib/search.ts [hunk 1]
```diff
@@ -85,7 +85,25 @@ export function queryRemoteOCPAppResources(): IRequestResult<ISearchResult> {
...
+    return postRequest<SearchQuery, ISearchResult>(getBackendUrl() + apiSearchUrl, {
+        operationName: 'searchResult',
+        variables: {
+            input: [
+                {
+                    filters: [
+                        {
+                            property: 'kind',
+                            values: 'kustomization',
+                        },
```
Comment (chenz4027) [inline frontend/src/lib/search.ts:106]: I wonder if it's possible to filter label by its key, tried with regex /xxxx=*/ but failed... not sure if that's supported. For now I used filter in later code to filter resources with specific labels

Comment (chenz4027) [reply frontend/src/lib/search.ts:106]: Ah I see, thanks Feng! Guess we will have to it manually later then

**Review by fxiang1 — APPROVED** [review]
LGTM! Thanks Maggie!

**Pull request merged**